polaris.shopify.com/pages/examples/tooltip-icon-mode.tsx

```diff
@@ -0,0 +1,63 @@
+import {Tooltip, ButtonGroup, Button} from '@shopify/polaris';
+import React from 'react';
+import {withPolarisExample} from '../../src/components/PolarisExampleWrapper';
+
+function TooltipExample() {
+  return (
+    <div style={{padding: '75px 0'}}>
+      <ButtonGroup segmented fullWidth>
+        <Tooltip
+          content="Bold"
+          mode="icon"
```

Comment:
Is icon mode a remnant left over from the initial approach? Should we rename this file to demo the new prop? 

```suggestion
```